searchapp/static/student_view.js

```diff
@@ -188,22 +188,207 @@ function upload_click(e) {
 			data: formData,
 			headers: { "X-CSRFToken": csrftoken, crossOrigin: false},
 			success: function(d) {
```

Comment:
Whenever you see some practice practice like naming variables 'd', try to change those. I'm sure we have a bunch of those around. A good amount of refactoring can be done on the existing code so do as and when you can.